### ISSUE TITLE
Dasharo/docs: "301f28f" rev feedback thread (06-DEC-2021)

### ISSUE THREAD

**Issue by mikebdp2** [Dasharo/dasharo-issues#34]
1. [Intro](https://github.com/Dasharo/docs/blob/master/docs/index.md) homepage - I believe this should have a "Success Stories"-like section, which can show off the already-released PCs with Dasharo firmware provided by 3mdeb. And each further SKU section - could link and/or describe these successful products.
2. [Dasharo Workstation](https://github.com/Dasharo/docs/blob/master/docs/variants/workstation.md) - the first SKU page linked from a homepage - only contains a TBD. Could add a text that explains the need for secure workstation, how Dasharo can provide this, and describe a real example of already-released workstation product.
3. [Dasharo Firewall](https://github.com/Dasharo/docs/blob/master/docs/variants/firewall.md) - should be easy to replace at least the table's TBDs with the shop links etc.
4. [Dasharo Secure Firewall](https://github.com/Dasharo/docs/blob/master/docs/variants/secure-firewall.md) - Can describe these 4 TBD sections at Dasharo Firewall page above, and there - is enough to just highlight, what "Dasharo Secure Firewall" is additionally giving on top of a "regular" Firewall product.
5. [Ecosystem](https://github.com/Dasharo/docs/blob/master/docs/ecosystem/overview.md) - [OSF Community](https://github.com/Dasharo/docs/blob/master/docs/ecosystem/osf-community.md) page is empty, could at least copy-paste this part from [Intro](https://github.com/Dasharo/docs/blob/master/docs/index.md)
6. [Quality Principles](https://github.com/Dasharo/docs/blob/master/docs/quality-principles/introduction.md) - the links at Compliance section are broken? _(regarding the Markdown formatting)_
7. [Dasharo Quality Criteria](https://github.com/Dasharo/docs/blob/master/docs/quality-principles/dasharo-quality-criteria.md) and [Firmware Release Process](https://github.com/Dasharo/docs/blob/master/docs/osf-trolling-list/release_process.md) pages, linked from the left side's glossary of [docs.dasharo.com](https://docs.dasharo.com/), are not found.
8. [Value Proposition](https://github.com/Dasharo/docs/blob/master/docs/value-prop/overview.md) - don't know if we should link "New Golden Age" to Youtube there, this may distract the reader. The content should stay within [docs.dasharo.com](https://docs.dasharo.com/) when possible.
9. [Nix Dev Env](https://github.com/Dasharo/docs/blob/master/docs/dev-env/nix-dev-env.md) - "Docker containers considered harmful" is already described at [Firmware Build Process](https://github.com/Dasharo/docs/blob/master/docs/osf-trolling-list/build_process.md)
10. [Dasharo Custom](https://github.com/Dasharo/docs/blob/master/docs/variants/custom.md) section hasn't been described at [Intro](https://github.com/Dasharo/docs/blob/master/docs/index.md) with the other SKUs.

**Comment by pietrushnic** [comment]
@mikebdp2 there is a lot of outadated stuff that I have not time to work on, but I appreciate your feedback. To comment above:

> 1. [Intro](https://github.com/Dasharo/docs/blob/master/docs/index.md) homepage - I believe this should have a "Success Stories"-like section, which can show off the already-released PCs with Dasharo firmware provided by 3mdeb. And each further SKU section - could link and/or describe these successful products.

In to the point, we need success story are you up to write something about KGPE-D16, PC Engines or OpenPOWER?
We need high quality case study, so maybe we can find some journalist who will contribute.

What would you say to polish that description and after placing hardware in 3mdeb shop start campaign for Dasharo firmware for that hardware?

> 2. [Dasharo Workstation](https://github.com/Dasharo/docs/blob/master/docs/variants/workstation.md) - the first SKU page linked from a homepage - only contains a TBD. Could add a text that explains the need for secure workstation, how Dasharo can provide this, and describe a real example of already-released workstation product.

I think I introduced SKUs to early. There should be differentiation maybe on higher level. We did quite granular differentiation, but most of it is visible at system level through firmware, payload/bootloader or OS features. My current view is to get rid of SKUs and rely purely on exact hardware and great cases study, if pool of supported hardware will grow we may see common features for some segments and with the business model behind that we can reintroduce SKUs, or whatever the name we will came with.

> 3. [Dasharo Firewall](https://github.com/Dasharo/docs/blob/master/docs/variants/firewall.md) - should be easy to replace at least the table's TBDs with the shop links etc.

a/a

> 4. [Dasharo Secure Firewall](https://github.com/Dasharo/docs/blob/master/docs/variants/secure-firewall.md) - Can describe these 4 TBD sections at Dasharo Firewall page above, and there - is enough to just highlight, what "Dasharo Secure Firewall" is additionally giving on top of a "regular" Firewall product.

I would move that to given case study/success story.

> 5. [Ecosystem](https://github.com/Dasharo/docs/blob/master/docs/ecosystem/overview.md) - [OSF Community](https://github.com/Dasharo/docs/blob/master/docs/ecosystem/osf-community.md) page is empty, could at least copy-paste this part from [Intro](https://github.com/Dasharo/docs/blob/master/docs/index.md)

I have also some concerns and improvements to that section. I'm thinking we should present Dasharo in light of recent sustainability problems with old AGESA ports.

Dasharo is open source firmware distribution, we prefer clean and simple code, long term maintenance, privacy-respecting implementation,  liberty for the owners, and trustworthiness for all.

> 6. [Quality Principles](https://github.com/Dasharo/docs/blob/master/docs/quality-principles/introduction.md) - the links at Compliance section are broken? _(regarding the Markdown formatting)_

Rather not linked to anything. I should elaboreate on every principle and do really good writing about that. `ENOTIME`.
Definitely something I have to fix and this is the reason why we should split this issue in long run.

> 7. [Dasharo Quality Criteria](https://github.com/Dasharo/docs/blob/master/docs/quality-principles/dasharo-quality-criteria.md) and [Firmware Release Process](https://github.com/Dasharo/docs/blob/master/docs/osf-trolling-list/release_process.md) pages, linked from the left side's glossary of [docs.dasharo.com](https://docs.dasharo.com/), are not found.

a/a

> 8. [Value Proposition](https://github.com/Dasharo/docs/blob/master/docs/value-prop/overview.md) - don't know if we should link "New Golden Age" to Youtube there, this may distract the reader. The content should stay within [docs.dasharo.com](https://docs.dasharo.com/) when possible.

Agree, do you think building in YT player is OK? There are very little presentation which are so important. I can think about one more lecture which is to the point: https://www.usenix.org/conference/osdi21/presentation/fri-keynote

> 9. [Nix Dev Env](https://github.com/Dasharo/docs/blob/master/docs/dev-env/nix-dev-env.md) - "Docker containers considered harmful" is already described at [Firmware Build Process](https://github.com/Dasharo/docs/blob/master/docs/osf-trolling-list/build_process.md)

I have to write about https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/tools/misc/coreboot-toolchain
and its value to reproducible toolchain and reproducible build environment.

> 10. [Dasharo Custom](https://github.com/Dasharo/docs/blob/master/docs/variants/custom.md) section hasn't been described at [Intro](https://github.com/Dasharo/docs/blob/master/docs/index.md) with the other SKUs.

Since I want to get rid of SKUs I will just add set of supported hardware.

**Comment by pietrushnic** [comment]
Another way of looking at Dasharo is Firmware as a Service.

**Comment by mikebdp2** [comment]
@pietrushnic 
> **1.** In to the point, we need success story are you up to write something about KGPE-D16, PC Engines or OpenPOWER?

Yes, I can try to write such a story in brief, then you could expand it _(since you may know enthralling behind-the-scenes)_ 

> **2.** I think I introduced SKUs to early.

Maybe present a feature table, which will user-friendly highlight the feature differences between the SKUs. Then, a description of SKUs may be shortened: describe the "basic SKU" and then what the "advanced SKUs" are giving on top of it.

> **5.** I have also some concerns and improvements to that section. I'm thinking we should present Dasharo in light of recent sustainability problems with old AGESA ports. Dasharo is open source firmware distribution, we prefer clean and simple code, long term maintenance, privacy-respecting implementation, liberty for the owners, and trustworthiness for all.

This is a great idea, since it may show what Dasharo can give on top of "just a coreboot". Although, I wonder if there would be an option to use Dasharo with SeaBIOS: SeaBIOS is so small in size and supports unique features like the virtual floppy loading.

> **8.** Agree, do you think building in YT player is OK?

Ideally, may put this video either on your own server and embed it using the opensource player, or - if it's too large - maybe to some external free file hosting which supports the embedding linking of videos without ads. _If you'd like, I could try finding it._

> **9.** Another way of looking at Dasharo is Firmware as a Service: https://news.ycombinator.com/item?id=18720195

Problem with "Firmware as a Service" label - is that some people may think that's a subscription model where the things break _(stop booting?)_ if you don't pay X $ in Y time.

> Firmware as a Service optimizes UEFI and other system firmware for timely quality patches that keep firmware up to date and enables efficient development of post-launch features

"Maintained Firmware" or "Prolonged Maintenance Firmware" sounds better and also stays more close to that description.
